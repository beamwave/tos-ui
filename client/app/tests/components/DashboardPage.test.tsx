// import React, { Component } from 'react'
// import { shallow } from 'enzyme'
// import { DashboardPage } from '../../components/DashboardPage'

// test('should render DashboardPage component', () => {
//   // const filters = {
//   //   medium: 'medium'
//   // }

//   const wrapper = shallow(<DashboardPage />)
//   expect(wrapper).toMatchSnapshot()
// })
