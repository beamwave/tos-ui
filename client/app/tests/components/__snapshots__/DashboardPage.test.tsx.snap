// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage component 1`] = `
<main
  className="dashboard-page"
>
  <div>
    <section
      className="image-results"
    >
      <Gallery
        backdropClosesModal={false}
        currentImage={0}
        enableImageSelection={false}
        enableKeyboardInput={true}
        enableLightbox={true}
        id="ReactGridGallery"
        imageCountSeparator=" of "
        images={
          Array [
            Object {
              "src": undefined,
              "thumbnail": undefined,
              "thumbnailHeight": undefined,
              "thumbnailWidth": undefined,
            },
            Object {
              "src": undefined,
              "thumbnail": undefined,
              "thumbnailHeight": undefined,
              "thumbnailWidth": undefined,
            },
            Object {
              "src": undefined,
              "thumbnail": undefined,
              "thumbnailHeight": undefined,
              "thumbnailWidth": undefined,
            },
          ]
        }
        isOpen={false}
        lightboxWidth={1024}
        margin={2}
        preloadNextImage={true}
        rowHeight={180}
        showCloseButton={true}
        showImageCount={true}
        showLightboxThumbnails={false}
      />
    </section>
    <aside
      className="primary-config"
    >
      <main>
        <form
          className="filter-container"
        >
          <label
            htmlFor="select"
          >
            medium
          </label>
          <div
            className="angle-down"
          >
            <FontAwesomeIcon
              border={false}
              className="icon"
              fixedWidth={false}
              flip={null}
              icon="angle-down"
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size={null}
              spin={false}
              symbol={false}
              transform={null}
            />
          </div>
          <select
            className="select"
            name="select"
            value="art"
          >
            <option
              value="art"
            >
              art
            </option>
            <option
              value="comments"
            >
              comments
            </option>
            <option
              value="clothes"
            >
              clothes
            </option>
            <option
              value="people"
            >
              people
            </option>
          </select>
        </form>
        <form
          className="filter-container"
        >
          <label
            htmlFor="select"
          >
            device
          </label>
          <div
            className="angle-down"
          >
            <FontAwesomeIcon
              border={false}
              className="icon"
              fixedWidth={false}
              flip={null}
              icon="angle-down"
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size={null}
              spin={false}
              symbol={false}
              transform={null}
            />
          </div>
          <select
            className="select"
            name="select"
            value="art"
          >
            <option
              value="art"
            >
              art
            </option>
            <option
              value="comments"
            >
              comments
            </option>
            <option
              value="clothes"
            >
              clothes
            </option>
            <option
              value="people"
            >
              people
            </option>
          </select>
        </form>
        <form
          className="filter-container"
        >
          <label
            htmlFor="select"
          >
            project
          </label>
          <div
            className="angle-down"
          >
            <FontAwesomeIcon
              border={false}
              className="icon"
              fixedWidth={false}
              flip={null}
              icon="angle-down"
              inverse={false}
              listItem={false}
              mask={null}
              pull={null}
              pulse={false}
              rotation={null}
              size={null}
              spin={false}
              symbol={false}
              transform={null}
            />
          </div>
          <select
            className="select"
            name="select"
            value="art"
          >
            <option
              value="art"
            >
              art
            </option>
            <option
              value="comments"
            >
              comments
            </option>
            <option
              value="clothes"
            >
              clothes
            </option>
            <option
              value="people"
            >
              people
            </option>
          </select>
        </form>
      </main>
    </aside>
    <aside
      className="secondary-config"
    >
      <main>
        <div
          className="filter-group"
        >
          <div
            className="input-group"
          >
            <label
              htmlFor="color"
            >
              color
            </label>
            <input
              className="input"
              name="color"
              placeholder="#hex or name"
              type="text"
            />
            <div
              className="color-group"
            >
              <div
                className="color-1"
              />
              <div
                className="color-2"
              />
              <div
                className="color-3"
              />
              <FontAwesomeIcon
                border={false}
                className="icon"
                fixedWidth={false}
                flip={null}
                icon="angle-right"
                inverse={false}
                listItem={false}
                mask={null}
                pull={null}
                pulse={false}
                rotation={null}
                size={null}
                spin={false}
                symbol={false}
                transform={null}
              />
            </div>
          </div>
        </div>
        <div
          className="filter-group"
        >
          <div
            className="input-group"
          >
            <label
              htmlFor="length"
            >
              length
            </label>
            <input
              name="length"
              placeholder="px"
              type="text"
            />
          </div>
          <div
            className="input-group"
          >
            <label
              htmlFor="width"
            >
              width
            </label>
            <input
              name="width"
              placeholder="px"
              type="text"
            />
          </div>
        </div>
      </main>
    </aside>
  </div>
</main>
`;
